server/src/models/about-transactions.ts

```diff
@@ -0,0 +1,10 @@
+import mongoose from '../mongo-db.js'
+import { MODEL_ABOUT_TRANSACTIONS } from './constants'
+
+export const aboutTransactionsSchema = new mongoose.Schema({
+  count_txs: Number,
+  amount_txs: Number,
+  date: String,
```

Comment:
index?key?

Reply:
added, date is index, because date has format DD-MM-YYYY